### PR TITLE
FAI-7999: Support passing component group when fetching component uptimes

### PR DESCRIPTION
**Pull request by chalenge**
## Description

- Support passing component group when fetching component uptimes
- Fixes an issue where how we are writing application platform name in uptimes doesn't match how we are writing application from components as [we prepend the group name](https://github.com/faros-ai/airbyte-connectors/blob/main/destinations/airbyte-faros-destination/src/converters/statuspage/common.ts#L188)

## Type of change
- [X] Bug fix
- [ ] New feature
- [ ] Breaking change

## Related issues

> Fix [#1]() 

## Migration notes

> Describe migration notes if any

## Extra info

> Add any additional information

### CODE REVIEW

**Review by matiaslcoulougian — APPROVED** [review]
LGTM

**Review by ypc-faros — APPROVED** [review]

File: sources/statuspage-source/test/index.test.ts [hunk 1]
```diff
@@ -493,7 +509,51 @@ describe('index', () => {
         pageId: 'n3wb7hf336hn',
         componentId: 'component2',
         startDate: '2023-05-22',
+        componentGroupId: 'string',
```
Comment (ypc-faros) [inline sources/statuspage-source/test/index.test.ts:512]: nit: Maybe use a better value than `string`

Comment (chalenge) [reply sources/statuspage-source/test/index.test.ts:512]: Sadly this is coming from the test file JSON, so would have to change multiple things :-(

Comment (ypc-faros) [reply sources/statuspage-source/test/index.test.ts:512]: No worries 👍

**Pull request merged**